packages/block-library/src/media-text/edit.native.js

```diff
@@ -154,25 +190,39 @@ class MediaTextEdit extends Component {
 			mediaWidth,
 			imageFill,
 			focalPoint,
+			verticalAlignment,
 		} = attributes;
+		const mediaAreaWidth =
+			( mediaWidth && ! shouldStack
+				? ( containerWidth * mediaWidth ) / 100
+				: containerWidth ) - styles.mediaAreaPadding.width;
```

Comment:
I'm not sure that we need to subtract the `styles.mediaAreaPadding.width` here. See that the margin is bigger than before in the media-text block.
<img src="https://user-images.githubusercontent.com/16336501/84027372-28507380-a98f-11ea-8f57-3f031b87f25f.png" alt="Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-08 at 13 18 36" width="350"/>

Reply:
Nice catch! So we don't need this when it's `stacked` but when it's in landscape I do need to subtract the margin. I changed the code so it works this way without affecting when it's in Portrait.